client/state/sites/selectors.js

```diff
@@ -102,3 +102,24 @@ export function getSiteSlug( state, siteId ) {
 export function isRequestingSites( state ) {
 	return !! state.sites.fetchingItems.all;
 }
+
+/**
+ * Returns true if the current user has the specified capability for the site,
+ * false if the user does not have the capability, or null if the capability
+ * cannot be determined (if the site is not currently known).
+ *
+ * @see https://codex.wordpress.org/Function_Reference/current_user_can
+ *
+ * @param  {Object}   state      Global state tree
+ * @param  {Number}   siteId     Site ID
+ * @param  {String}   capability Capability label
+ * @return {?Boolean}            Whether current user has capability
+ */
+export function isCurrentUserCapableForSite( state, siteId, capability ) {
```

Comment:
>What about `currentUserCan()` placed in the currentUser state subtree instead?

This goes back to an issue of confused relation between sites and capabilities, where the [sites response includes capabilities for the current user](https://developer.wordpress.com/docs/api/1.1/get/me/sites/). From the state perspective, I'd argue it makes sense to keep here as it's selecting on the sites reducer state.

Though, perhaps we should consider including a new reducer in `currentUser` to track these capabilities as sites are received from the API?